canton/BUILD.bazel

```diff
@@ -1519,15 +1517,16 @@ genrule(
         "@go_googleapis//google/rpc:status.proto",
         ":ledger-api-protos-fg",
     ],
-    outs = ["Com/Daml/Ledger/Api/" + b for b in ledger_api_haskellpb_sources],
+    outs = ["Com/Daml/Ledger/Api/V2/PackageService.hs"],
     cmd = """
         for src in $(locations :ledger-api-protos-fg); do
             $(location @proto3-suite//:compile-proto-file) \
                 --includeDir """ + google_protobuf_src + """ \
                 --includeDir """ + google_rpc_src + """ \
                 --includeDir """ + ledger_api_proto_source_root + """ \
-                --proto $${src#*community/ledger-api/src/main/protobuf/} \
+                --proto com/daml/ledger/api/v2/$$(basename $$src) \
                 --out $(@D)
+            cp -ruv $(@D)/Com/Daml/Ledger/Api/V2/* $(@D)
```

Comment:
Nasty workaround.
Here I had a problem to write it in a better way - somehow I was always getting hs files generated in a wrong folder.
Someone with a better knowledge of BAZEL + protogen here is probably needed.